base/strings/util.jl

```diff
@@ -49,9 +49,6 @@ endswith(str::AbstractString, chars::Chars) = !isempty(str) && last(str) in char
 startswith(a::String, b::String) = sizeof(a) ≥ sizeof(b) &&
     ccall(:memcmp, Int32, (Ptr{UInt8}, Ptr{UInt8}, UInt), a, b, sizeof(b)) == 0
 
-startswith(a::Vector{UInt8}, b::Vector{UInt8}) = length(a) ≥ length(b) &&
```

Comment:
Wouldn't this need a deprecation? Also unrelated to `missing`.

Reply:
Yeah, maybe. Though since it's not documented and it only works for `Vector{UInt8}`, is it really needed? I can add one, or move this to another PR, as you prefer.